modules/api/main.tf

```diff
@@ -396,28 +404,30 @@ resource "aws_ecs_task_definition" "api_task_definition" {
        "privileged": false,
        "name": "${local.app_name}",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         },
-         {
-           "name": "APP_NAME",
-           "value": "${local.app_name}"
-         }
+         ${local.ecs_shared_env_vars}
        ],
-       "logConfiguration": {
-         "logDriver": "awslogs",
-         "options": {
-           "awslogs-region": "${var.region}",
-           "awslogs-group": "${var.log_group}",
-           "awslogs-stream-prefix": "${local.app_name}"
-         }
-       }
+     ${module.logging_configs.log_configuration}
      }
    ]
-
 DEFINITION
 
+  depends_on = [
+    module.logging_configs
+  ]
+
+}
+
+module "logging_configs" {
+  source            = "../logging_configs"
+  app_name          = local.app_name
+  environment_name  = var.environment_name
```

Comment:
Oh yeah I see what @rraub is talking about. Could we just switch this out for terraform.workspace for consistency for now?

Reply:
Switched 👍